core/unit_test/TestGraph.hpp

```diff
@@ -329,32 +329,36 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), when_all_cycle) {
   //----------------------------------------
 }
 
-// This test is disabled because we don't currently support copying to host,
+// This test requires that the graph execution space can access
+// the host memoy space because we don't currently support copying to host,
 // even asynchronously. We _may_ want to do that eventually?
-TEST_F(TEST_CATEGORY_FIXTURE(graph), DISABLED_repeat_chain) {
-  auto graph = Kokkos::Experimental::create_graph(
-      ex, [&, count_host = count_host](auto root) {
-        //----------------------------------------
```

Comment:
There doesn't seem to be a reason for changing these lines and below.

Reply:
What do you mean ?

Reply:
No, you deleted a bunch of lines like
```
//----------------------------------------
```
which just made the diff bigger (ignoring whitespace changes) than necessary.

Reply:
To me these lines are useless and make the code harder to read (I guess people would use that to expand/collapse that region, but in general it's just distracting). But if you want them back, I'll do.

Reply:
Yeah, let's just put them back since you have to fix the `skip` variable anyway.